src/clims/services/workflow/camunda/client.py

```diff
@@ -204,6 +217,14 @@ def is_relative(name):
                 call_activity.attrib[
                     "calledElement"] = called_process_id_fully_qualified
 
+        # Qualify all form keys
```

Comment:
According to my taste, I prefer much shorter methods than this. I had to scroll to find out where "namespace" comes from (input argument).

Reply:
I agree! I can refactor it when I have time. It's often more clear when reading it than when writing it.